packages/neos-ui/src/Containers/Modals/ReloginDialog/style.css

```diff
@@ -0,0 +1,12 @@
+.dialog {
+    max-width: 400px;
```

Comment:
I think it would be better to add smaller size of modal to Dialog component as a prop. We might need those smaller components in the future, and it's better to be consistent.